site/content/en/docs/manual/advanced/automatic-annotation.md

```diff
@@ -2,54 +2,106 @@
+to automatically pre-annotate your data with pre-trained models.
 
-1. To launch automatic annotation, you should open the dashboard and find a task which you want to annotate.
-   Then click the `Actions` button and choose option `Automatic Annotation` from the dropdown menu.
+You can also **add models** from popular platforms
+like **Hugging Face** or **Roboflow** to expand the
+range of available options.
+
+> Note: Automatic Annotation is a [paid CVAT Cloud feature](https://www.cvat.ai/pricing/cloud).
+
+See:
+
+- [Running Automatic annotation](#running-automatic-annotation)
+- [Labels matching](#labels-matching)
+- [Models](#models)
+- [Adding models from Hugging Face and Roboflow](#adding-models-from-hugging-face-and-roboflow)
+
+## Running Automatic annotation
+
+To start automatic annotation, do the following:
+
+1. On the top menu, click **Tasks**.
+2. Find the task you want to annotate and click **Action** > **Automatic Annotation**.
 
    ![](/images/image119_detrac.jpg)
 
-1. In the dialog window select a model you need. DL models are created for specific labels, e.g.
-   the Crossroad model was taught using footage from cameras located above the highway and it is best to
-   use this model for the tasks with similar camera angles.
-   If it's necessary select the `Clean old annotations` checkbox.
-   Adjust the labels so that the task labels will correspond to the labels of the DL model.
-   For example, let’s consider a task where you have to annotate labels “car” and “person”.
-   You should connect the “person” label from the model to the “person” label in the task.
-   As for the “car” label, you should choose the most fitting label available in the model - the “vehicle” label.
-   If the chosen model supports automatic attributes detection
-   (like facial expressions, for example: ``serverless/openvino/omz/intel/face-detection-0205``),
-   you can also map attributes between the DL model and your task.
-   The task requires to annotate cars only and choosing the “vehicle” label implies annotation of all vehicles,
-   in this case using auto annotation will help you complete the task faster.
-   Click `Submit` to begin the automatic annotation process.
+3. In the Automatic annotation dialog, from the drop-down list, select [model](#models).
+4. [Match labels](#labels-matching) of the model and the task.
+5. (Optional) In case you need the model to return masks as polygons, switch toggle **Return masks as polygons**.
+6. (Optional) In case you need to remove all previous annotations, switch toggle **Clean old annotations**.
 
    ![](/images/image120.jpg)
 
-1. At runtime - you can see the percentage of completion.
-   You can cancel the automatic annotation by clicking on the `Cancel`button.
+7. Click **Annotate**.
+
+CVAT will show the progress of annotation on the progress bar.
+
+![Progress bar](/images/image121_detrac.jpg)
+
+You can stop the automatic at any moment by clicking **Cancel**.
```

Comment:
```suggestion
You can stop the automatic annotation at any moment by clicking **Cancel**.
```

Also, I don't think you can say "click Cancel" if it doesn't actually say "Cancel" on the screen.